packages/platform-bun/src/BunHttpTest.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @since 0.41.3
```

Comment:
If you keep all the since tags at 1.0.0, as we are not yet api stable.

Reply:
I'll change it. I noticed Giulio uses precise `@since` versions in the /schema and I think it's beneficial for the consumers of the lib.